samples/core/src/main/resources/db/migration/postgre/V0001__R001_Create_schema_pg.sql.pg

```diff
@@ -0,0 +1,135 @@
+-- This is the SQL script for setting up the DDL for the PostGres database
```

Comment:
Please name the folder `postgres` instead of `postgre` as this will only cause confusions and typo problems.